docs/apache-hive-catalog.md

```diff
@@ -28,11 +28,13 @@ The Hive catalog supports creating, updating, and deleting databases and tables
-| `client.pool-size`  | The maximum number of Hive metastore clients in the pool for Gravitino.                                                                                                                                           | 1             | No       | 0.2.0         |
-| `gravitino.bypass.` | Property name with this prefix passed down to the underlying HMS client for use. Such as `gravitino.bypass.hive.metastore.failure.retries = 3` indicate 3 times of retries upon failure of Thrift metastore calls | (none)        | No       | 0.2.0         |
+| Property Name                            | Description                                                                                                                                                                                                                                                                      | Default Value | Required | Since Version |
+|------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `metastore.uris`                         | The Hive metastore service URIs, separate multiple addresses with commas. Such as `thrift://127.0.0.1:9083`                                                                                                                                                                      | (none)        | Yes      | 0.2.0         |
+| `client.pool-size`                       | The maximum number of Hive metastore clients in the pool for Gravitino.                                                                                                                                                                                                          | 1             | No       | 0.2.0         |
+| `gravitino.bypass.`                      | Property name with this prefix passed down to the underlying HMS client for use. Such as `gravitino.bypass.hive.metastore.failure.retries = 3` indicate 3 times of retries upon failure of Thrift metastore calls                                                                | (none)        | No       | 0.2.0         |
+| `client.pool.cache.eviction-interval-ms` | The cache pool eviction interval.                                                                                                                                                                                                                                                |               | No       | 0.4.0         |
```

Comment:
The default value of `client.pool.cache.eviction-interval-ms` is empty.

Reply:
Fixed.